src/Support/DatabaseFactoryHelper.php

```diff
@@ -19,6 +19,7 @@ public function __construct(
 	public function resetResolvers(): void
 	{
 		$this->unsetProperty(Factory::class, 'modelNameResolver');
+        $this->unsetProperty(Factory::class, 'modelNameResolvers');
 		$this->unsetProperty(Factory::class, 'factoryNameResolver');
```

Comment:
Probably should just call the new `Factory::flushState();` method here.

Reply:
I believe that `::flushState()` didn't exist before the change to `$modelNameResolvers`, which would cause issues for anyone on an earlier version of Laravel.

Reply:
```php
if (version_compare(\Illuminate\Foundation\Application::class, '11.43.0', '>=')) {
    Factory::flushState();
} else {
    // use old code
}
```

Reply:
@crynobone I've updated this PR, but also sent a [framework PR](https://github.com/laravel/framework/pull/54736) to deprecate the original property so that I (hopefully) don't have to stick with `v11.41.3` until this PR is merged and tagged.